jupyterlab_celltests/tests.py

```diff
@@ -78,6 +84,9 @@ def writeout_test(fp, cells):
             to_write.append(INDENT + 'pass')
 
         fp.writelines(to_write)
+        fp.write(INDENT*2 + '""")\n')
+        fp.write(INDENT*2 + 'self.kernel.await_reply(msg_id)\n')
```

Comment:
I think you would also want to process messages after this until the status = idle message is received. This is only awaiting a reply that is has received the execute input, and started processing it.